packages/wonder-blocks-badge/src/components/badge.tsx

```diff
@@ -165,8 +165,13 @@ const styles = StyleSheet.create({
         color: badgeTokens.root.color.foreground,
     },
     icon: {
-        width: badgeTokens.icon.sizing.width,
-        height: badgeTokens.icon.sizing.height,
+        // Use minWidth and minHeight to ensure custom sized icons don't
```

Comment:
I created https://khanacademy.atlassian.net/browse/WB-1954 for tracking work to support the new Icon component in other WB components